### ISSUE TITLE
Can't use alias to create index

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#8614]
#### Steps/Code to reproduce:
```
import random

from pymilvus import (
    connections, list_collections,
    FieldSchema, CollectionSchema, DataType,
    Collection
)


def hello_milvus():
    # create connection
    connections.connect()

    print(f"\nList collections...")
    print(list_collections())

    # create collection
    dim = 128
    default_fields = [
        FieldSchema(name="count", dtype=DataType.INT64, is_primary=True),
        FieldSchema(name="random_value", dtype=DataType.DOUBLE),
        FieldSchema(name="float_vector", dtype=DataType.FLOAT_VECTOR, dim=dim)
    ]
    default_schema = CollectionSchema(fields=default_fields, description="test collection")

    print(f"\nCreate collection...")
    collection = Collection(name="hello_milvus", schema=default_schema)
    collection.create_alias("a")
    alias = Collection("a")
    print(f"\nList collections...")
    print(list_collections())

    #  insert data
    nb = 3000
    vectors = [[random.random() for _ in range(dim)] for _ in range(nb)]
    alias.insert(
        [
            [i for i in range(nb)],
            [float(random.randrange(-20, -10)) for _ in range(nb)],
            vectors
        ]
    )

    print(f"\nGet collection entities...")
    print(collection.num_entities)


    print(f"\n Get alias entities")
    print(alias.num_entities)
    # create index
    default_index = {"index_type": "IVF_FLAT", "params": {"nlist": 128}, "metric_type": "L2"}
    print(f"\nCreate index...")
    alias.create_index(field_name="float_vector", index_params=default_index)
hello_milvus()
```

#### Expected result:
Create index successfully

#### Actual results:
Create index failed

#### Environment:
- Milvus version(e.g. v2.0.0-RC2 or 8b23a93): master-20210926-8091e76
- Deployment mode(standalone or cluster): standalone 
- SDK version(e.g. pymilvus v2.0.0rc2):2.0.0rc7.dev13
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others: 

#### Additional context:
```
List collections...
[]

Create collection...

List collections...
['hello_milvus']

Get collection entities...
3000

alias entities
3000

Create index...
Error: <BaseException: (code=1, message=CreateIndex failed, error = collection a not found)>
Traceback (most recent call last):
  File "hello_milvus.py", line 108, in <module>
    hello_milvus()
  File "hello_milvus.py", line 65, in hello_milvus
    alias.create_index(field_name="float_vector", index_params=default_index)
  File "/Users/wxzhu/opt/anaconda3/lib/python3.8/site-packages/pymilvus/orm/collection.py", line 1025, in create_index
    return conn.create_index(self._name, field_name, index_params,
  File "/Users/wxzhu/opt/anaconda3/lib/python3.8/site-packages/pymilvus/client/stub.py", line 61, in handler
    raise e
  File "/Users/wxzhu/opt/anaconda3/lib/python3.8/site-packages/pymilvus/client/stub.py", line 45, in handler
    return func(self, *args, **kwargs)
  File "/Users/wxzhu/opt/anaconda3/lib/python3.8/site-packages/pymilvus/client/stub.py", line 834, in create_index
    return handler.create_index(collection_name, field_name, params, timeout, **kwargs)
  File "/Users/wxzhu/opt/anaconda3/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 67, in handler
    raise e
  File "/Users/wxzhu/opt/anaconda3/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 59, in handler
    return func(self, *args, **kwargs)
  File "/Users/wxzhu/opt/anaconda3/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 780, in create_index
    raise BaseException(status.error_code, status.reason)
pymilvus.client.exceptions.BaseException: <BaseException: (code=1, message=CreateIndex failed, error = collection a noåå
```

**Comment by zhuwenxing** [comment]
@lsgrep Please take a look

**Comment by lsgrep** [comment]
Ok

**Comment by yhmo** [comment]
@zhuwenxing Already fixed by #8780, please verify it in latest build.

**Comment by zhuwenxing** [comment]
Verified and passed
